### ISSUE TITLE
Launch screen looks empty

### ISSUE THREAD

**Issue by skyylex** [seanoshea/BluePlaquesLondon#38]
##### I think we could use appIcon image to fill empty space and to emphasize project style. Something like this:

<a href="http://tinypic.com?ref=15hh1rq" target="_blank"><img src="http://i59.tinypic.com/15hh1rq.png" border="0" alt="Image and video hosting by TinyPic"></a>

**Comment by seanoshea** [comment]
I'm not sure the current version of the Launch Image really fits in with Apple's guidelines at https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/LaunchImages.html. Perhaps I should just have a static image with a tab bar at the bottom?

**Comment by skyylex** [comment]
You are right. I've checked a few apps like Facebook, Viber, Skype and some of iOS native apps, all of them use something like a screenshot of initial screen cleaned from all of the controls and content. Using this as a stand point we need a tab bar without tabs. One thing, I'm not sure about the main color. Blue is the color of the icon. But whole UI is made with white and gray colors.

**Comment by seanoshea** [comment]
Yep - completely agree. We'd need icons for a few different device sizes as well (iPad/iPhone 5/6/6+). I like your idea of a basic gray color. Not sure how your photoshop skills are for creating these types of images, but I could give it a shot.
